doc/modules/mod_event_pusher.md

```diff
@@ -45,4 +45,4 @@ Refer to their specific documentation to learn more about their functions and co
 [sns]: ./mod_event_pusher_sns.md
 [push]: ./mod_event_pusher_push.md
 [http_notification]: ./mod_event_pusher_http.md
-[rabbit]: ./mod_event_pusher_rabbit.md
\ No newline at end of file
+[rabbit]: ./mod_event_pusher_rabbit.md
```

Comment:
What is the difference here?

Reply:
It's a new line added at the end of file, I don't know why GH fails to show it clearly.